sycl/test/basic_tests/vec_convert.cpp

```diff
@@ -78,9 +78,9 @@ int main() {
   test<float, float, 8, rounding_mode::automatic>(
       float8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f},
       float8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f});
-  test<float, half, 8, rounding_mode::automatic>(
-      float8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f},
-      half8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f});
+  test<double, float, 8, rounding_mode::automatic>(
+      double8{+2.3, +2.5, +2.7, -2.3, -2.5, -2.7, 0., 0.},
+      float8{+2.3f, +2.5f, +2.7f, -2.3f, -2.5f, -2.7f, 0.f, 0.f});
 
   // rte
   test<int, int, 8, rounding_mode::rte>(
```

Comment:
Since you are adding separate files for conversions between different types, I suggest you to leave in this file only `float` to `int` conversions - just to avoid testing the same combinations in different files

Reply:
Maybe I should rename this file then?

Reply:
> Maybe I should rename this file then?

Makes sense